interpolator test fails with recent update to pykrige 1.4.0

Output after upgrading from pykrige 1.3.1 to 1.4.0 on conda-forge

``` python
================================================================================= FAILURES =================================================================================
_____________________________________________________________ TestComlinkGridInterpolator.test_default_kriging _____________________________________________________________

self = <pycomlink.tests.test_interpolator.TestComlinkGridInterpolator testMethod=test_default_kriging>

    def test_default_kriging(self):
        cml_list = load_processed_cml_list()
    
        # TODO: Use 'C' backend when available via pip installed pykirge
        interpolator = pycml.spatial.interpolator.ComlinkGridInterpolator(
            cml_list=cml_list,
            variable='R',
            resolution=0.05,
            interpolator=(pycml.spatial.interpolator.
                          OrdinaryKrigingInterpolator(backend='loop')))
    
        ds = interpolator.loop_over_time()
    
        assert ds.R.isel(time=0).values.shape == (16, 23)
    
        # TODO: Check if theses results change when using the 'C' backend
        np.testing.assert_array_almost_equal(
            ds.R.isel(time=24).values[1:4, 1:4],
            np.array([[2.41876631, 3.55747763, 5.21045399],
                      [2.59474309, 3.74337047, 4.95187848],
>                     [2.89174467, 3.57024647, 4.49247846]]))
E       AssertionError: 
E       Arrays are not almost equal to 6 decimals
E       
E       (mismatch 100.0%)
E        x: array([[ 2.312077,  3.591382,  5.370885],
E              [ 2.410231,  3.689951,  4.975776],
E              [ 2.661749,  3.460712,  4.415959]])
E        y: array([[ 2.418766,  3.557478,  5.210454],
E              [ 2.594743,  3.74337 ,  4.951878],
E              [ 2.891745,  3.570246,  4.492478]])

pycomlink/tests/test_interpolator.py:121: AssertionError
--------------------------------------------------------------------------- Captured stdout call ---------------------------------------------------------------------------
75 CMLs read in
--------------------------------------------------------------------------- Captured stderr call ---------------------------------------------------------------------------
100%|██████████| 48/48 [00:01<00:00, 27.37it/s]
============================================================================= warnings summary =============================================================================
pycomlink/tests/test_comlink_channel.py::TestComlinkChannelAppendData::test_append_max_age
  /Users/chwala-c/code/pycomlink/pycomlink/core/comlink_channel.py:259: SettingWithCopyWarning: 
  A value is trying to be set on a copy of a slice from a DataFrame.
  Try using .loc[row_indexer,col_indexer] = value instead
  
  See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
    df['txrx'] = df.tx - df.rx

-- Docs: http://doc.pytest.org/en/latest/warnings.html
============================================================= 1 failed, 33 passed, 1 warnings in 20.08 seconds =============================================================
```

--- a/pycomlink/tests/test_interpolator.py
+++ b/pycomlink/tests/test_interpolator.py
@@ -114,11 +114,15 @@ class TestComlinkGridInterpolator(unittest.TestCase):
         assert ds.R.isel(time=0).values.shape == (16, 23)
 
         # TODO: Check if theses results change when using the 'C' backend
+        #
+        # Note, that the reference values had to be updated after switching
+        # from pykrige version 1.3.1 to version 1.4.0. See this PR
+        # https://github.com/bsmurphy/PyKrige/issues/110
         np.testing.assert_array_almost_equal(
             ds.R.isel(time=24).values[1:4, 1:4],
-            np.array([[2.41876631, 3.55747763, 5.21045399],
-                      [2.59474309, 3.74337047, 4.95187848],
-                      [2.89174467, 3.57024647, 4.49247846]]))
+            np.array([[2.312077, 3.591382, 5.370885],
+                      [2.410231, 3.689951, 4.975776],
+                      [2.661749, 3.460712, 4.415959]]))
 
 
 class TestGetDataFrameForCmlVariable(unittest.TestCase):

